src/app/content/highlights/components/EditCard.tsx

```diff
@@ -71,6 +79,18 @@ const EditCard = React.forwardRef<HTMLElement, Props>((
     }
   };
 
+  const cancelEditing = React.useCallback(() => {
+    setPendingAnnotation(defaultAnnotation());
+    setEditing(false);
+    onCancel();
+  }, [defaultAnnotation, onCancel]);
+
+  React.useEffect(() => {
+    if (!isFocused) {
+      cancelEditing();
+    }
+  }, [isFocused, cancelEditing]);
```

Comment:
don't provide `cancelEditing` as a dependency. if the linter complains disable it.